File: src/main/java/com/bot/utils/CommandCategories.java
package com.bot.utils;

import com.jagrosh.jdautilities.command.Command;

public class CommandCategories {

    public final static Command.Category VOICE = new Command.Category("voice");
    public final static Command.Category GENERAL = new Command.Category("general");
    public final static Command.Category NSFW = new Command.Category("nsfw");
    public final static Command.Category MODERATION = new Command.Category("moderation");

    // Derivatives of General Category for more granularity
    public final static Command.Category REDDIT = new Command.Category("reddit");
    public final static Command.Category MEME = new Command.Category("meme");


}

File: src/main/java/com/bot/utils/CommandCategories.kt
package com.bot.utils

import com.jagrosh.jdautilities.command.Command

object CommandCategories {
    @JvmField
    val VOICE: Command.Category = Command.Category("voice")
    @JvmField
    val GENERAL: Command.Category = Command.Category("general")
    @JvmField
    val NSFW: Command.Category = Command.Category("nsfw")
    @JvmField
    val MODERATION: Command.Category = Command.Category("moderation")

    // Derivatives of General Category for more granularity
    @JvmField
    val REDDIT: Command.Category = Command.Category("reddit")
    val MEME: Command.Category = Command.Category("meme")
    val SUBSCRIPTION: Command.Category = Command.Category("subscription")

    @JvmField
    val OWNER: Command.Category = Command.Category("owner")
}
